doc/source/user_guide/meshing_workflow/meshing_workflows.rst

```diff
@@ -572,6 +572,220 @@ Switch to solution mode
 
     solver = meshing.switch_to_solver()
 
+
+2D meshing workflow
+-------------------
+Use the **2D** meshing workflow to mesh specific two dimensional geometries.
+The following example shows you how to use the 2D Meshing workflow.
+
+Import geometry
+~~~~~~~~~~~~~~~
+
+.. code:: python
+
+    import ansys.fluent.core as pyfluent
+    from ansys.fluent.core import examples
+
+    import_file_name = examples.download_file('NACA0012.fmd', 'pyfluent/airfoils')
+    meshing = pyfluent.launch_fluent(
+        mode="meshing", precision='double', processor_count=2
+    )
+    meshing.workflow.InitializeWorkflow(WorkflowType="2D Meshing")
+    meshing.workflow.TaskObject["Load CAD Geometry"].Arguments.set_state(
+        {
+            r"FileName": import_file_name,
+            r"LengthUnit": r"mm",
+            r"Refaceting": {
+                r"Refacet": False,
+            },
+        }
+    )
+    meshing.workflow.TaskObject["Load CAD Geometry"].Execute()
```

Comment:
For all these descriptions in the quotation marks, can they be edited to use sentence case and concluding punctuation? I'm not sure if they are message strings shown to the user. If so, they should be written as a sentence that concludes with a period.

Reply:
These are basically display names of tasks by Fluent. These should be as it is. Thank you :)